package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dashjs",
-  "version": "3.1.3",
+  "version": "0.0.0",
```

Comment:
I don't think we would want to change this to 0.0.0? Current version of dash.js in development branch is 4.7.3

Reply:
Sorry, I pushed this code to the wrong repo by mistake.  Closing the PR.